File: script/go/println/fizz_buzz_println.go
package main

import (
    "flag"
	"strconv"
    "fmt"
    // "time"
)

func main() {
    flag.Parse()
    n, _ := strconv.Atoi(flag.Args()[0])

	// start := time.Now()
    for i := 1; i <= n; i++ {
        switch {
        case i%15 == 0:
            fmt.Printf("fizzbuzz\n")
        case i%3 == 0:
            fmt.Printf("fizz\n")
        case i%5 == 0:
            fmt.Printf("buzz\n")
        default:
            fmt.Printf("%d", i)
        }
    }
    // fmt.Printf("elapsed_time:%.4f[sec]\n", time.Now().Sub(start).Seconds())
}

File: script/dart/println/fizz_buzz_println.dart
// import 'dart:async';

main(List<String> args) {
    int n = int.parse(args[0]);
    // Stopwatch s = Stopwatch();

    // s.start();
    for (var i = 1; i <= n; i++) {
        if (i % 15 == 0) {
            print("fizzbuzz");
        } else if (i % 3 == 0) {
            print("fizz");
        } else if (i % 5 == 0) {
            print("buzz");
        } else {
            print(i);
        }
    }
    // var time = s.elapsed;
    // print("elapsed_time:${time.inMilliseconds/1000}[sec]");
}
